### PR TITLE
Echai full stack dapp

### PR DESCRIPTION
**Pull request by DevSwayam**
#### Project Description :
Echai full stack web app using React, ethers.js and hardhat for testing.
   
#### FEATURES: 
A user can donate fund worth of a chai amount to developers(owner) who deployed the site with a message attached to it.
A user can view how many funds the dapp holds.
A user can see whole list of donators from start to latest up to date with their message and funds.

#### Text Stack used:
[x] Solidity
[x] Ethers
[x] React
[x] HardHat
[x] remix

### UI Screenshot:
image

### CODE REVIEW

**Comment by Vikash-8090-Yadav** [comment]
There is background noise..Fix thatv

**Comment by DevSwayam** [comment]
Should I remove drive link or just remove background voice and update the link?

**Comment by Vikash-8090-Yadav** [comment]
@DevSwayam can u please do one thing , instead of posting  the video on the drive . paste  the video directly on the readne.md of the chai project

ps :  u   have also not created  the readme for this project  please do that .

**Comment by DevSwayam** [comment]
Ok

**Comment by DevSwayam** [comment]
Bro i am done with everything now.

**Comment by Vikash-8090-Yadav** [comment]
@DevSwayam  please change this : 
1. Your video contain the sound    , create a no sound video of the  running site not of  the  code . Hope u get this . 
2. Explain the proper steps  for `HOW TO RUN THIS PROJECT ON YOUR PC:`
3. Improve the readme of this , for refrence take  help from the merged one .

**Comment by DevSwayam** [comment]
Bro i have old pc i cant turn on my recording in live running of npm hence i did it from start and i laso shown the features :
Their is list of donators. with their messages and i think my recoding get blocked whenever metamask ask for password.

I am improving readme.md

File: Level2/Echai_funding/README.md [hunk 1]
```diff
@@ -42,12 +42,17 @@ https://drive.google.com/drive/folders/1fM9g84aN4GYZfJp8QN-ps5xertshxx-V?usp=sha
...
+ 
  step 3: setup your own metamask walleet address and alchemy api key via .env file.
- step: test smart contract with already written test cases.
- step4: if you face any issue feel free to dm me GitHub = @DevSwayam
  
+ step4: Deploy and test smart contract with already written test cases.
+ 
+ step5: After go to terminal and cd client and run your react app now your good to see transactions.
+  If you face any issue feel free to dm me GitHub = @DevSwayam
+
```
Comment (Vikash-8090-Yadav) [inline Level2/Echai_funding/README.md:54]: remove the 52 line and below

File: Level2/Echai_funding/README.md [hunk 2]
```diff
@@ -42,12 +42,17 @@ https://drive.google.com/drive/folders/1fM9g84aN4GYZfJp8QN-ps5xertshxx-V?usp=sha
...
+ step 1: clone the repo.
+ 
+ step 2: install hardhat (link =https://hardhat.org/hardhat-runner/docs/getting-started#overview) and react (link= https://reactjs.org/docs/getting-started.html) package according to documentation.
+ 
  step 3: setup your own metamask walleet address and alchemy api key via .env file.
- step: test smart contract with already written test cases.
- step4: if you face any issue feel free to dm me GitHub = @DevSwayam
  
+ step4: Deploy and test smart contract with already written test cases.
+
```
Comment (Vikash-8090-Yadav) [inline Level2/Echai_funding/README.md:50]: after step4 write 
do `npm install` to install all dependencies

Comment (DevSwayam) [reply Level2/Echai_funding/README.md:50]: Bro I already told to reffer hardhat and react documentation for installation it's already included in it

File: Level2/Echai_funding/README.md [hunk 3]
```diff
@@ -7,16 +7,49 @@ all recieved funds get listed below for transparancy and past records.
...
-This project demonstrates a basic Hardhat use case. It comes with a sample contract, a test for that contract, and a script that deploys that contract.
+Echai full stack web app using React, ethers.js and hardhat for testing.
+
+A decentralized application using hardhat and ethers.js and will be deployed on netlify server.
+
+Anyone can donate funds to trust with a message attached to it.
+all recieved funds get listed below for transparancy and past records.
+All can see total amount of funds Website owner holds.
+
+#### Text Stack used:
```
Comment (Vikash-8090-Yadav) [inline Level2/Echai_funding/README.md:20]: write like this 
<ul>
<li>
[x] Solidity
<li>
[x] Ethers
</li>
<li>
[x] React
</li>
<li>
[x] HardHat
</li>
<li>
[x] remix
</li>
</ul>

File: Level2/Echai_funding/README.md (hunk 1 above)
Comment (Vikash-8090-Yadav) [reply Level2/Echai_funding/README.md:59]: Please do this ...

File: Level2/Echai_funding/README.md (hunk 2 above)
Comment (Vikash-8090-Yadav) [reply Level2/Echai_funding/README.md:54]: 'npm install' is used to dowoad the dependencies that is being used by your project . if someone doesn't know about all this then by doing this all drpendecies will installed . hope u get this .

**Comment by Vikash-8090-Yadav** [comment]
Change the video bro , u r not showing the running live site . 
Also write in proper way for the installation process .

File: Level2/Echai_funding/README.md (hunk 2 above)
Comment (DevSwayam) [reply Level2/Echai_funding/README.md:54]: Check I have included

File: Level2/Echai_funding/README.md (hunk 1 above)
Comment (DevSwayam) [reply Level2/Echai_funding/README.md:59]: Bro check updated files I have up
Screenshot_2023-01-12-17-41-56-67_40deb401b9ffe8e1df2f1cc5ba480b12
dated it

**Comment by Vikash-8090-Yadav** [comment]
They way u written is not good bro .It's in level2 so maitain the format of readme . When any beginner see ur code then it's hard to understand him .Make all thngs in point's .
am talking about the installation steps.
and check  the previous comment also .

**Comment by Vikash-8090-Yadav** [comment]
please add a video which shows the live  of the site . u added the video of code only

**Comment by DevSwayam** [comment]
Now please check.

**Comment by Vikash-8090-Yadav** [comment]
Great Looking forward too see more contribution from your side .

**Comment by DevSwayam** [comment]
> Great Looking forward too see more contribution from your side .

Thank you so much

**Pull request merged**